copy/copy.go

```diff
@@ -615,13 +613,16 @@ func (c *copier) copyConfig(ctx context.Context, src types.Image) error {
 		if err != nil {
 			return errors.Wrapf(err, "Error reading config blob %s", srcInfo.Digest)
 		}
-		bar := c.createProgressBar(srcInfo, "config")
-		defer bar.Finish()
-		bar.Start()
+
+		// make the short digest only 10 characters long to make it align with the blob output
+		bar := c.createProgressBar(srcInfo, shortDigest(srcInfo.Digest)[0:10], "config")
 		destInfo, err := c.copyBlobFromStream(ctx, bytes.NewReader(configBlob), srcInfo, nil, false, true, bar)
 		if err != nil {
 			return err
 		}
+		if bar != nil {
+			bar.SetTotal(0, true)
```

Comment:
Should this be 0 or `len(configBlob)`?

Reply:
0 is intended. It has no impact on the displayed total size but it will trigger the bar to be completed.